PANGOLIN-3680: data table settings menu (#2797)
commercetools/ui-kit: .changeset/slimy-kangaroos-hammer.md

---
'@commercetools-uikit/data-table-manager': minor
---

> feat(data-table-settings): added new dropDownMenu to dataTableManager

Reviewer: ```suggestion
replace the dataTableManager SelectInput with the new dropDownMenu component
```
